ara/api/migrations/0009_distincthost.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.19 on 2021-04-03 17:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0008_playbook_controller'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DistinctHost',
+            fields=[
+                ('name', models.CharField(max_length=255, primary_key=True, serialize=False)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('updated', models.DateTimeField(auto_now=True)),
+                ('latest_host', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='api.Host')),
+            ],
+            options={
+                'db_table': 'distinct_hosts',
+            },
+        ),
+    ]
```

Comment:
This create the DistinctHost model -- maybe we could also add a migration step that "computes" the current distinct hosts and inserts the rows ? That way, after the migration is done, the list is already populated and ready to be used.

We have some python code running in this migration as an example: https://github.com/ansible-community/ara/blob/master/ara/api/migrations/0004_duration_in_database.py

Reply:
I was curious and tried something that seems to work: https://gist.github.com/dmsimard/d4fea5ce3aff720f706270d37b9632d4

Reply:
This is an awesome approach, I haven't thought about that, but really like it. 

Regarding your question:
> TODO: Should we sort by another column ?

I would say the updated column is fine because this will be also the criteria for future data (as it is implemented right now).

Reply:
I've added you proposal as well to this PR branch